lib/timeline/Core.js

```diff
@@ -1044,6 +1045,11 @@ class Core {
       direction: contentsOverflow ? Hammer.DIRECTION_ALL : Hammer.DIRECTION_HORIZONTAL
     });
 
+    // set the long press time
+    this.hammer.get('press').set({
+      time: this.options.multiselectLongPressTime
```

Comment:
I think you're missing default value here.

Reply:
Ok, where should I define the default value?

Reply:
The default value should now be set.